Maestro/subscriptions.json

```diff
@@ -6,11 +6,11 @@
       "vsoProject": "DevDiv",
       "buildDefinitionId": 1084
     },
-    // A 'build' that mirrors corefx github into VSO
-    "corefx-mirror": {
+    // A 'build' that mirrors changes from github into VSTS
+    "github-vsts-mirror": {
       "vsoInstance": "devdiv.visualstudio.com",
       "vsoProject": "DevDiv",
-      "buildDefinitionId": 1119
+      "buildDefinitionId": 1308884
```

Comment:
This is a build id, not a definition id--should be `8289`.

Reply:
Oops thanks. I thought that number looked a little high.